packages/plugins/documentation/server/routes/index.js

```diff
@@ -47,6 +47,15 @@ module.exports = [
       ],
     },
   },
+  {
+    method: 'GET',
+    path: '/v:major(\\d+).:minor(\\d+).:patch(\\d+)/download',
+    handler: 'documentation.download',
+    config: {
+      auth: false,
+      middlewares: [restrictAccess],
+    },
+  },
```

Comment:
Should I use a new endpoint or should I change the implementation on the default route (on documentation) and add some `if (req.accepts('json')) return ctx.send(json);` ?